cmd/nix_init.go

```diff
@@ -0,0 +1,240 @@
+package cmd
+
+/*	License: GPLv3
+	Authors:
+		Mirko Brombin <send@mirko.pm>
+		Pietro di Caprio <pietro@fabricators.ltd>
+	Copyright: 2023
+	Description: Apx is a wrapper around multiple package managers to install packages and run commands inside a managed container.
+*/
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"html/template"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+type UnitData struct {
+	User string
+}
+
+func NewNixInitCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "init",
+		Short: "Initialize nix repository",
+		RunE:  initNix,
+	}
+
+	return cmd
+}
+func initNix(cmd *cobra.Command, args []string) error {
```

Comment:
I think the logic part should go inside `core`, maybe something like `core/nix.go`

Reply:
yes, when it's not `Draft` anymore I'll do lots of cleanup. Agreed.